pkg/driver/node_server.go

```diff
@@ -1715,7 +1715,14 @@ func (driver *Driver) NodeExpandVolume(ctx context.Context, request *csi.NodeExp
 		// figure out if volumePath is actually a staging path
 		stagedDevice, err := readStagedDeviceInfo(request.GetVolumePath())
 		if err != nil {
-			return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Cannot get staging device info from volume path %s", err.Error()))
+			log.Tracef("Error retrieving stagedDevice from volume path %v",request.GetVolumePath())
```

Comment:
Can this be 'log.Infof' or error log instead of trace?

Reply:
I removed that logger and merged the error message.